assets/tests/test_permissions.py

```diff
@@ -165,68 +272,44 @@ def has_object_permission(self, obj):
             self.perm.has_object_permission(self.request, None, obj)
         )
 
+    def tearDown(self):
+        cache.delete(f"{self.user.username}:lookup")
 
-class OrPermissionTests(TestCase):
 
-    def test_view_perms(self):
-        """Test all combinations of two OR'd classes for the view permissions"""
-        cases = (
-            # 0 = A.has_permission(), 1 = B.has_permission(), 2 = expected result
-            (False, False, False),
-            (False, True, True),
-            (True, False, True),
-            (True, True, True),
-        )
-        for case in cases:
-            class A(BasePermission):
-                def has_permission(self, request, view):
-                    return case[0]
+class UserInIARGroupPermissionTests(TestCase):
 
-            class B(BasePermission):
-                def has_permission(self, request, view):
-                    return case[1]
-            self.assertEqual(permissions.OrPermission(A, B)().has_permission(None, None), case[2])
+    def setUp(self):
+        # Create a mock incoming request and view
+        self.request = Request(HttpRequest())
```

Comment:
Do we want to make use of [APIRequestFactory](http://www.django-rest-framework.org/api-guide/testing/#creating-test-requests) here?